src/common/Requirements.ts

```diff
@@ -163,16 +178,25 @@ export abstract class BaseSetup implements RequirementList {
             tree.insert(setupMessage);
             const rootNode = tree.nodes[setupMessage];
             testResult.tests.forEach((test) => {
-                const message = `${test.testResult} (${test.duration.toFixed(
```

Comment:
I think we should rework this part. I think we should print out `<title> (duration): Passed/Failed` followed by extra info for the Passed/Failed scenario (if any). we can further discuss as a team on what we'd like to include in the output, but that's my opinion on this matter.

Reply:
For now I have left the format as `<PASSED or FAILED>: <Title> (0.000 sec)`

Reply:
Sure sounds good. I wonder if it'd be better to print the duration after Passed/Failed (so that it would be in a tabular format and easier to read/find the tasks that take a long time): `<PASSED/FAILED> (1.234 sec): <TITLE>`.  Just a thought.